pkg/installer/custominstallconfig.go

```diff
@@ -108,6 +108,7 @@ func (m *manager) applyInstallConfigCustomisations(installConfig *installconfig.
 		if err != nil {
 			return nil, err
 		}
+
```

Comment:
Might want to remove this superfluous change?

Reply:
Thanks, I thought I had got that one :smile: